src/test/java/pl/coderslab/service/DriverServiceTest.java

```diff
@@ -0,0 +1,111 @@
+package pl.coderslab.service;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.runners.MockitoJUnitRunner;
+import pl.coderslab.model.Cargo;
+import pl.coderslab.model.Driver;
+import pl.coderslab.repository.DriverRepository;
+
+import javax.validation.constraints.AssertTrue;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.Assert.*;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@RunWith(MockitoJUnitRunner.class)
+public class DriverServiceTest {
+
+    @Mock
+    private DriverRepository driverRepository;
+
+    @InjectMocks
+    private DriverService driverService;
+
+    @Test
+    public void shouldAdd() {
+
+        Driver driver = new Driver();
+        driver.setId(1L);
+        driver.setFirstName("Adam");
+        driver.setLastName("Klim");
+        driver.setPhoneNumber("123123123");
+        driver.setSalary(3000.0);
+
+        given(driverRepository.findById(driver.getId())).willReturn(Optional.of(driver));
+
+        driverService.add(driver);
+        Driver driver1 = driverService.showById(1L).orElseThrow();
```

Comment:
driver1 to nie jest dobra nazwa, driverFromService, persistedDriver, addedDriver będzie napenwo lepsze